impl_core/src/main/java/io/opencensus/implcore/trace/export/SpanExporterImpl.java

```diff
@@ -154,6 +254,7 @@ private void onBatchExport(List<SpanData> spanDataList) {
     private Worker(int bufferSize, Duration scheduleDelay) {
       spans = new ArrayList<>(bufferSize);
       this.bufferSize = bufferSize;
+      this.maxReferencedSpans = 10L * bufferSize;
```

Comment:
Any specific reason we chose this value? Do we want to make it configurable?

Reply:
I was wondering the same.

Reply:
Added some explanation about why I decided to go with 4 * bufferSize in the end.